test/powershell/Language/Parser/TypeAccelerator.Tests.ps1

```diff
@@ -22,4 +416,14 @@ Describe "Type accelerators" -Tags "CI" {
         $TypeAcceleratorsType::Remove('msft_2174855_rm')
         $TypeAcceleratorsType::Get['msft_2174855_rm'] | Should Be $null
     }
+
+    It 'Should have a type acclerator for: <Accelerator>' -TestCases $TypeAcceleratorTestCases {
+        param($Accelerator, $Type)
+        $TypeAcceleratorsType::Get[$Accelerator] | Should be ($Type)

```

Comment:
This did not work as expected. The object returned by the type accelerator get is System.RuntimeType.

```powershell
$TypeAcceleratorsType = [psobject].Assembly.GetType("System.Management.Automation.TypeAccelerators")
$TypeAcceleratorsType::Get['String']

$TypeAcceleratorsType::Get['String'] | Get-Member
```